__tests__/index.test.js

```diff
@@ -138,4 +139,56 @@ describe('GET /api/articles', () => {
       expect(datesInMs).toBeSorted({descending: true})
     })
   })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+  it('should return all comments on one article in an array', () => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then((res) => {
+      const comments = res.body.comments
+      expect(Array.isArray(comments)).toBe(true)
+      expect(comments.length).toBe(11)
+      comments.forEach((article) => {
+        expect(article).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          votes: expect.any(Number),
+          author: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
Here we want to check the article_id is 1 specifically, to ensure we are getting the correct article's comments back